OrbitCore/TracepointInfoManager.cpp

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "TracepointInfoManager.h"
+
+bool TracepointInfoManager::AddUniqueTracepointEventInfo(
```

Comment:
I don't really have context about what you are doing but just wondering why you return a boolean when this is called in the listener without checking what the method returns.

Reply:
This is intended to be useful in the long-term for different checks. If you look at the way it is done for the string manager or the callstacks, the mechanism is similar so, I also wanted to keep it consistent with the rest of the data retrieved from the collector.